gourmet/exporters/exporter.py

```diff
@@ -1,13 +1,12 @@
+from gi.repository import GLib
+from gi.repository import Pango
```

Comment:
As a style thing, I'd generally leave the `gi` imports below the standard library ones. But I don't feel strongly about that.

Reply:
done